tests/plotting/test_plotting.py

```diff
@@ -122,1635 +122,1697 @@ def verify_cache_image(plotter):
                       f'{error}')
 
 
-@skip_no_plotting
```

Comment:
Umm, I think something went off the rails here. I'll take a look once we agree this is the state I should be reviewing :sweat_smile:

Reply:
I commented a ton of tests out and never enabled them again.  Fixed in https://github.com/pyvista/pyvista/pull/1247/commits/5236f56b9be8014f17b0ba7132a751111c9677c0

Reply:
Are you aware that you can run specific tests with things like `pytest -v -k test_plot_shadows`? Just making sure.